#
# Fixes the invisible cursor color when using GTKSpell and a black
# input box (when "Use the text box font & colors" is ON).
#
diff -u -r1.66 fe-gtk.c
--- xchat-2.6.4/src/fe-gtk/fe-gtk.c	7 Jun 2006 06:16:51 -0000	1.66
+++ xchat-2.6.4p1/src/fe-gtk/fe-gtk.c	12 Jun 2006 03:13:24 -0000
@@ -225,7 +225,11 @@
 const char cursor_color_rc[] =
 	"style \"xc-ib-st\""
 	"{"
+#ifdef USE_GTKSPELL
+		"GtkTextView::cursor-color=\"#%02x%02x%02x\""
+#else
 		"GtkEntry::cursor-color=\"#%02x%02x%02x\""
+#endif
 	"}"
 	"widget \"*.xchat-inputbox\" style : application \"xc-ib-st\"";
 
